src/components/transactionPropTypes.js

```diff
@@ -67,6 +67,17 @@ export default PropTypes.shape({
         }),
     ),
 
+    /** Selected participants */
+    participants: PropTypes.arrayOf(
```

Comment:
Should this be required? And should the property isPolicyExpenseChat also be required?

Reply:
@cead22 in the new TS type
https://github.com/Expensify/App/blob/cc0e82d713d8242ce68e35ea47ba9fbc3d3a25c2/src/types/onyx/Transaction.ts#L136

https://github.com/Expensify/App/blob/cc0e82d713d8242ce68e35ea47ba9fbc3d3a25c2/src/types/onyx/IOU.ts#L9

These fields are not required. `participants` exists after the user selects participant so why do you think these fields should be required?

What do you think about adding a fallback value for `transaction.participant ` here
```
    const isMerchantRequired = _.some(transaction.participants, (participant) => Boolean(participant.isPolicyExpenseChat));
```

Reply:
>why do you think these fields should be required?

What you said makes sense, I only thought maybe they could be required since we're using them. 

>why do you think these fields should be required?

We can, and it would make it more obvious that `isPolicyExpenseChat` isn't a required prop, but at the same time, the result is the same because `participant.isPolicyExpenseChat` will be cast to bool even without the explicit cast, so I don't feel strongly either way